fahrschein/src/main/java/org/zalando/fahrschein/ProblemHandlingRequest.java

```diff
@@ -113,15 +113,15 @@ private void handleAuthError(final JsonNode rootNode, final int statusCode) thro
 
     private void handleBatchItemResponse(JsonNode rootNode) throws IOException {
         final BatchItemResponse[] responses = objectMapper.treeToValue(rootNode, BatchItemResponse[].class);
-        final List<BatchItemResponse> failed = new ArrayList<>(responses.length);
+        boolean partiallyFailed = false;
         for (BatchItemResponse batchItemResponse : responses) {
-            if (batchItemResponse.getPublishingStatus() != BatchItemResponse.PublishingStatus.SUBMITTED) {
-                failed.add(batchItemResponse);
+            if(batchItemResponse.getPublishingStatus() != BatchItemResponse.PublishingStatus.SUBMITTED) {
```

Comment:
lets make it explicit

```suggestion
            if(batchItemResponse.getPublishingStatus() == BatchItemResponse.PublishingStatus.FAILED ||
                batchItemResponse.getPublishingStatus() == BatchItemResponse.PublishingStatus.ABORTED) {
```